packages/framework/src/Resources/views/Admin/Form/orderItems.html.twig

```diff
@@ -2,82 +2,84 @@
-                                </th>
-                                <th class="text-right">{{ 'Unit price excluding VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th class="text-right">{{ 'Total including VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th class="text-right">{{ 'Total excluding VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th></th>
-                        </tr>
-                        </thead>
-                        <tbody
-                            id="js-order-items"
-                            data-prototype="{{ orderItemMacro.orderItem(form.itemsWithoutTransportAndPayment.vars.prototype, null, null)|escape }}"
-                            data-order-product-add-url="{{ url('admin_order_addproduct', { orderId: order.id }) }}"
-                        >
-                            {% for productItem in order.productItems %}
-                                {{ orderItemMacro.orderItem(form.itemsWithoutTransportAndPayment[productItem.id], productItem.id, productItem) }}
-                            {% endfor %}
+        <div class="form-group">
+            <div class="form-group__title">{{ 'Order items'|trans }}</div>
+            <div class="form-line">
 
-                            {% for orderItemId, orderItemForm in form.itemsWithoutTransportAndPayment %}
-                                {% if not orderItemForm.rendered %}
-                                    {{ orderItemMacro.orderItem(orderItemForm, orderItemId, null) }}
-                                {% endif %}
-                            {% endfor %}
+                    <div class="table-touch-always">
+                        {{ form_errors(form.itemsWithoutTransportAndPayment) }}
+                        <div class="table-touch-always__in">
+                            <table class="table-main table-main--with-inline-form">
+                                <thead>
+                                    <tr>
+                                        <th><span class="padding-left-10">{{ 'Name'|trans }}</span></th>
+                                        <th class="text-right">{{ 'Catalogue number'|trans }}</th>
+                                        <th class="text-right"><span class="table-main__column-with-info">{{ 'Unit price including VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</span></th>
+                                        <th class="text-right">{{ 'Amount'|trans }}</th>
+                                        <th class="text-right">{{ 'Unit'|trans }}</th>
+                                        <th class="text-right">{{ 'VAT rate (%)'|trans }}</th>
+                                        <th class="text-center">
+                                            <span class="display-inline-block" style="width: 80px">
```

Comment:
Could not we write this inline style in different way?

Reply:
fixed by helper

Reply:
Hello @topolanekmartin, for me is this point done, what do you think?

Reply:
Hello @pesektomas, @TomasGottvald I think it is ok now.